device/nokia/x86_64-nokia_ixr7250e_36x400g-r0/Nokia-IXR7250E-36x400G/0/jr2cp-nokia-18x400g-config.bcm

```diff
@@ -2055,3 +2055,4 @@ modreg IPS_FORCE_LOCAL_OR_FABRIC FORCE_FABRIC=1
 
 xflow_macsec_secure_chan_to_num_secure_assoc_encrypt=2
 xflow_macsec_secure_chan_to_num_secure_assoc_decrypt=4
+cmic_dma_abort_in_cold_boot=0
```

Comment:
Can you add this parameter to src/sonic-device-data/tests/permitted_list

Reply:
Judy, I have added to  src/sonic-device-data/tests/permitted_list. Please check